setup.py

```diff
@@ -43,9 +43,6 @@ def walk_subpkg(name):
     'folium.plugins'
 ]
 
-LICENSE = read('LICENSE.txt')
-long_description = '{}\n{}'.format(read('README.rst'), read('CHANGES.txt'))
```

Comment:
Could we still have the changelog following the readme?

Reply:
I can re-add that. I thought it was too verbose and boring for PyPI but I don't have strong opinions about it.

Reply:
Hmm yeah maybe. If someone wants to read the changelog they know where to find it I guess.

Reply:
The CHANGELOG is versioned so I guess that a link to the latest master version would make that easy to find and still keeping it simple. What do you think?

Reply:
That seems good.